content/en/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -1,34 +1,33 @@
 ---
 reviewers:
-- enisoc
-- erictune
-- foxish
-- janetkuo
-- kow3ns
+  - enisoc
+  - erictune
+  - foxish
+  - janetkuo
+  - kow3ns
 title: DaemonSet
 content_template: templates/concept
 weight: 50
 ---
 
 {{% capture overview %}}
 
-A _DaemonSet_ ensures that all (or some) Nodes run a copy of a Pod.  As nodes are added to the
-cluster, Pods are added to them.  As nodes are removed from the cluster, those Pods are garbage
-collected.  Deleting a DaemonSet will clean up the Pods it created.
+A _DaemonSet_ ensures that all (or some) Nodes run a copy of a Pod. As nodes are added to the
+cluster, Pods are added to them. As nodes are removed from the cluster, those Pods are garbage
+collected. Deleting a DaemonSet will clean up the Pods it created.
 
 Some typical uses of a DaemonSet are:
 
 - running a cluster storage daemon, such as `glusterd`, `ceph`, on each node.
 - running a logs collection daemon on every node, such as `fluentd` or `filebeat`.
-- running a node monitoring daemon on every node, such as [Prometheus Node Exporter](https://github.com/prometheus/node_exporter), [Flowmill](https://github.com/Flowmill/flowmill-k8s/), [Sysdig Agent](https://docs.sysdig.com), `collectd`, [Dynatrace OneAgent](https://www.dynatrace.com/technologies/kubernetes-monitoring/), [AppDynamics Agent](https://docs.appdynamics.com/display/CLOUD/Container+Visibility+with+Kubernetes), [Datadog agent](https://docs.datadoghq.com/agent/kubernetes/daemonset_setup/), [New Relic agent](https://docs.newrelic.com/docs/integrations/kubernetes-integration/installation/kubernetes-installation-configuration), Ganglia `gmond`, [Instana Agent](https://www.instana.com/supported-integrations/kubernetes-monitoring/) or [Elastic Metricbeat](https://www.elastic.co/guide/en/beats/metricbeat/current/running-on-kubernetes.html).
+- running a node monitoring daemon on every node, such as [Prometheus Node Exporter](https://github.com/prometheus/node_exporter), [Flowmill](https://github.com/Flowmill/flowmill-k8s/), [Sysdig Agent](https://docs.sysdig.com), `collectd`, [Dynatrace OneAgent](https://www.dynatrace.com/technologies/kubernetes-monitoring/), [AppDynamics Agent](https://www.appdynamics.com/solutions/cloud/cloud-monitoring/kubernetes-monitoring), [Datadog agent](https://docs.datadoghq.com/agent/kubernetes/daemonset_setup/), [New Relic agent](https://docs.newrelic.com/docs/integrations/kubernetes-integration/installation/kubernetes-installation-configuration), Ganglia `gmond`, [Instana Agent](https://www.instana.com/supported-integrations/kubernetes-monitoring/) or [Elastic Metricbeat](https://www.elastic.co/guide/en/beats/metricbeat/current/running-on-kubernetes.html).
```

Comment:
With the [content guide](https://kubernetes.io/docs/contribute/style/content-guide/) in mind, I suggest:
```suggestion
- running a cluster storage daemon on each node
- running a log collection daemon on every node
- running a host system monitoring tool on each node
```

Reply:
Hi Tim, 
I saw that too - let me look into why this is - I only changed one hyperlink using VSCode on Mac - but for some reason git thinks I changed a lot more lines... has to do with the editor I am assuming.

Can you leave this with me to investigate? Unless you know of issues around modifying daemonset.md file using VSCode on Mac....